src/components/Card.tsx

```diff
@@ -41,14 +17,33 @@ const Card: React.FC<IProps> = ({
   children,
   className,
 }) => (
-  <StyledCard className={className}>
-    <TopContent>
-      {emoji && <StyledEmoji size={3} text={emoji} />}
-      {title && <Title>{title}</Title>}
-      {description && <Description>{description}</Description>}
-    </TopContent>
+  <Flex
+    direction="column"
+    bg="ednBackground"
+    borderRadius="2px"
+    border="1px"
+    borderStyle="solid"
+    borderColor="lightBorder"
+    p={6}
+    className={className}
+  >
+    <div>
+      {emoji && (
+        <Emoji fontSize="5xl" text={emoji} mb={4} display="inline-block" />
```

Comment:
Emoji is already displayed as `inline-block`
```suggestion
        <Emoji fontSize="5xl" text={emoji} mb={4} />
```

Reply:
I'm not sure about this because if I don't force it, it's not there.
*I did my tests on this page https://ethereum.org/en/community/*

This one is when I force the `display: inline-block`:

![image](https://user-images.githubusercontent.com/43795504/189421272-54d267ae-13ab-441c-8a53-4c8010819d13.png)

And this one when I remove the `display` option:

![Screenshot from 2022-09-09 20-39-24](https://user-images.githubusercontent.com/43795504/189421372-1855c838-a52b-4326-8d7e-05a423a6fdd1.png)

Reply:
Interesting, I think I know what is going on.

In the `Emoji` component I've defined the display as `d="inline-block"` but the short `d` prop version is not supported any more in v2, so it is not taking effect.

Could you try changing that to `display="inline-block"` in the `Emoji` component? and test if that works?

Reply:
It's working :+1:

Reply:
Awesome, thanks!